packages/shared_preferences/shared_preferences/example/android/settings.gradle

```diff
@@ -10,6 +10,9 @@ if (pluginsFile.exists()) {
 
 plugins.each { name, path ->
     def pluginDirectory = flutterProjectRoot.resolve(path).resolve('android').toFile()
-    include ":$name"
-    project(":$name").projectDir = pluginDirectory
+    if (pluginDirectory.exists()) {
```

Comment:
Same issue with the minimum bound/potential Gradle issue here as with the macOS connectivity PR.

Reply:
Done